### PR TITLE
Feature/display past trips

### PR DESCRIPTION
**Pull request by soy-park**
I created a displayPastTrips() function where it iterates through the past trips as well as the destinations (by ID) via map(). For each trip, I added an innerHTML to the past trips section so that the interpolation will display the destination, travelers, dates, and durations for each past trip. 

I also added styling to each trip box in past trips. Specifically, I added a white background, border radius, and font color to each article of past trips.

@sarahcatherine311 @nicktu12

### CODE REVIEW

**Comment by soy-park** [comment]
Requesting review for this pull request @sarahcatherine311 @nicktu12

File: src/scripts.js [hunk 1]
```diff
@@ -17,14 +17,14 @@ window.addEventListener('load', loadHomePage);
...
         allTrips = new Trips(randomId, data[1].trips, data[2].destinations);
         allDestinations = new Destinations(data[2].destinations);
     })
     .then(() => {
         displayPastTrips();
-        displayUpcomingTrips();
+        // displayUpcomingTrips();
     })
     .catch(err => alert(err))
 }
```
Comment (sarahcatherine311) [inline src/scripts.js:30]: Nice work on your fetch!

File: src/scripts.js [hunk 2]
```diff
@@ -35,13 +35,16 @@ function generateRandomId() {
 
 function displayPastTrips() {
```
Comment (sarahcatherine311) [inline src/scripts.js:36]: This is wild! Good work! I used classes and methods for mine to dry it up a bit, but I'm very impressed by what all you accomplished in this one function.

**Review by sarahcatherine311 — APPROVED** [review]

**Comment by nicktu12** [comment]
👍🏻

**Pull request merged**